packages/dawn4py/package.py

```diff
@@ -30,3 +31,6 @@ class Dawn4py(PythonPackage):
 
     # will test that dawn4py is importable after the install
     import_modules = ['dawn4py']
+
+    def setup_environment(self, spack_env, run_env):
+        spack_env.set('CMAKE_ARGS', '-DLLVM_ROOT=' + self.spec['llvm'].prefix)
```

Comment:
```suggestion
        spack_env.set('LLVM_ROOT', self.spec['llvm'].prefix)
```

I don't think `CMAKE_ARGS` is intended for this purpose. We could instead use the `<Package>_ROOT` or `<Package>_DIR` environment variables.

https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure